gluten-ut/spark32/src/test/scala/org/apache/gluten/functions/ArithmeticAnsiValidateSuite.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gluten.functions
+
+import org.apache.gluten.config.GlutenConfig
+import org.apache.gluten.execution.ProjectExecTransformer
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.GlutenTestsTrait
+import org.apache.spark.sql.internal.SQLConf
+
+class ArithmeticAnsiValidateSuiteRasOff extends ArithmeticAnsiValidateSuite {
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf
+      .set(GlutenConfig.RAS_ENABLED.key, "false")
+  }
+}
+
+class ArithmeticAnsiValidateSuiteRasOn extends ArithmeticAnsiValidateSuite {
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf
+      .set(GlutenConfig.RAS_ENABLED.key, "true")
+  }
+}
+
+abstract class ArithmeticAnsiValidateSuite extends FunctionsValidateSuite with GlutenTestsTrait {
```

Comment:
It may be better to move the tests to the package below to reduce code duplication:
backends-velox/src/test/scala/org/apache/gluten/functions/

Reply:
You can use `testWithSpecificSparkVersion` to limit which Spark versions are used.